lib/webserver.js

```diff
@@ -300,7 +321,11 @@ WebServer.prototype = {
             let certificateList;
             if(userId && keyringName) {
               try {
-                certificateList = keyring_js.listKeyring(userId, keyringName);
+                let includingPrivateKey = false;
```

Comment:
add a note saying this will only work if there are truly only 2 types of objects in a keyring, but thus far that's all i've seen when working with users.